README.md

```diff
@@ -1,4 +1,4 @@
-[![Travis CI badge](https://api.travis-ci.com/rust-lang/rustc-dev-guide.svg?branch=master)](https://travis-ci.com/github/rust-lang/rustc-dev-guide)
+[![CI](https://github.com/rust-lang/rustc-dev-guide/actions/workflows/ci.yml/badge.svg)](https://github.com/rust-lang/rustc-dev-guide/actions/workflows/ci.yml)
```

Comment:
NOTE: This catches all the events, e.g., scheduled and push. Do we want to catch only one event?

Reply:
This shouldn't be affected by the date checker, and it should only be for events on the default branch, so I think it's fine. Why are we running this on a cron anyway?

Reply:
> Why are we running this on a cron anyway?

We can catch the broken links ASAP with the daily check.